### PR TITLE
Adding a font

### PR DESCRIPTION
**Pull request by BenjaminAleritsch**
Pull request includes:
- The folder: "font"
- The open type font file: "SBOL_2018.04.otf"
- The Inkscape svg-file that includes the symbols, pre-edited for FontForge: "InkscapeImport.svg"
- The FontForge project file: "SBOL_2018.04.sfd"
- Two example-files: "Example.odt" and "Example.pdf"
- A file with comments on installation, usage, maintenance and testing: "readme.txt"

### CODE REVIEW

File: font/readme.txt [hunk 1]
```diff
@@ -0,0 +1,118 @@
+How to install the font:
+
+  Linux(Ubuntu):
+  
+- Open font with Font Managemer and install on System-level
```
Comment (jakebeal) [inline font/readme.txt]: typo: "Manager"

File: font/readme.txt [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+How to use the font:
+
+- In text-processing program, insert "Special Character"
+
+- Choose required symbols (see Examples.pdf or Examples.odt for ideas)
+
+- Place and align according to demands
+
+- If e.g. a closed circular multiline plasmidmap is required, close vertical gaps adjusting line spacing by
```
Comment (jakebeal) [inline font/readme.txt]: e.g.,

File: font/readme.txt [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+  - clicking on menue: "Format" / "Paragraph" and
```
Comment (jakebeal) [inline font/readme.txt]: "menu"

File: font/readme.txt [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+  - select "Proportional" of ~"92%" at "Line Spacing"
```
Comment (jakebeal) [inline font/readme.txt]: Can you please explain why this is?

File: font/readme.txt [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  
+- After export to PDF, document can be imported to e.g. Inkscape via Poppler/Cairo import and used as path element
```
Comment (jakebeal) [inline font/readme.txt]: e.g.,  --> make this change for all uses

File: font/readme.txt [hunk 6]
```diff
@@ -0,0 +1,118 @@
+How to install the font:
```
Comment (jakebeal) [inline font/readme.txt]: Can you please convert this to a Markdown (.md) file?

**Review by jakebeal — CHANGES_REQUESTED** [review]
Minor edits requested for readme

**Comment by BenjaminAleritsch** [comment]
All changes done. I am not sure what more to comment on the line spacing option.

**Comment by jakebeal** [comment]
Thank you; I will approve the pull.  
Do please update the associated link in the readme when you've got your technical note uploaded.

**Comment by BenjaminAleritsch** [comment]
It will be the first thing I'll do after the note is published :) Thank you very much.

**Comment by jakebeal** [comment]
Thank you for all of the hard work making this font! This is a great contribution.

**Pull request merged**